assets/js/tagsinput.js

```diff
@@ -39,6 +42,84 @@ function setupTagsInput(tagBlock) {
     importTags();
   }
 
+  const ratingsTags = ['safe', 'suggestive', 'questionable', 'explicit', 'semi-grimdark', 'grimdark', 'grotesque'];
+  const checks = {numTags: false, ratingIncluded: false};
+
+  //TODO handle this better?
+  const errorsDisplayed = {numTags: false, ratingIncluded: false};
+
+  //chosing to pull from the plain editor field, even if fancy editor
+  //  is being displayed currently. This ensures the most current tags
+  //  will be pulled, in either case. This isn't using the tags[]
+  //  object, because it won't be up-to-date if the plain editor is being used
+  function tagsValid() {
+    const tagsArr = tagsTextareaPlain.value.split(',');
```

Comment:
This isn't very robust. You also need to trim spaces for the tag values, because it's perfectly legal to input tags separated with ", " (comma-space), and any number of spaces around each tag name.